frame/session/Cargo.toml

```diff
@@ -38,6 +38,8 @@ std = [
 	"codec/std",
 	"sp-std/std",
 	"frame-support/std",
+	"sp-core/std",
+	"sp-std/std",
```

Comment:
`sp-io` missing?

Reply:
Good catch, one of the two `sp-std/std` was supposed to be `sp-io`